src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, InfoPrinter {
+    private String figureName;
```

Comment:
We can use `getClass().getSimpleName()` where needed, lets remove removeNode variable